docs/content/quickstart.md

```diff
@@ -39,7 +39,7 @@ C:\Users\<username>\.workspacer\workspacer.config.csx
 
 workspacer is configured in the C# language, using types and interfaces provided by workspacer itself. this means that while workspacer requires knowledge of programming to use, it's configuration is extremely powerful, and allows you to modify, or even replace, most functionality that drives workspacer. for example, a common use case is to implement a custom layout engine (just define a class that satisfies the interface `ILayoutEngine`) entirely inside the config file, and then use it to define completely custom layout behavior, without needing to modify workspacer itself.
 
-workspacer's config file allows for __type checking__ and __intellisense__! currently, only VSCode with the C# extension is supported. opening the `.workspacer` folder in your home directory inside VSCode with the C# extension installed will provide syntax highlighting, type checking, and autocomplete automatically because of the `#r` directives at the top of the config file (so don't remove those, or else this won't work!).
+workspacer's config file allows for __type checking__ and __intellisense__! currently, only VSCode with the C# extension is supported. opening the `.workspacer` folder in your home directory inside VSCode with the C# extension installed will provide syntax highlighting, type checking, and autocomplete automatically because of the [`#r` directives at the top of the config file](https://github.com/workspacer/workspacer/blob/495ab72fdf80d8aedbdd790cc638e5f8e97a476e/src/workspacer/Config/workspacer.config.template.csx#L1) (so don't remove those, or else this won't work!).
```

Comment:
Should it be a permanent link?

Reply:
Would it be better to just reference the entire file then?

Reply:
I think so - the benefit would be that if the template is updated, then the link doesn't need to be updated.

On the other hand, if the template is moved, that'll break the link (hopefully this would be picked up in review) 🤞 .

Reply:
Ok, I'll change it